fast/stages/2-networking-c-nva/bgp-files/ew1b

```diff
@@ -0,0 +1,67 @@
+log syslog informational
```

Comment:
committed by mistake and no new line at the end of file

Reply:
sorry, removed!